### ISSUE TITLE
Remove functional requiremet 3 from SRS

### ISSUE THREAD

**Issue by gabrielasd** [gabrielasd/eomee#21]
Rather than verifying the symmetry of the input electron integrals, it might be better to use a helper function that transforms them from the molecular orbital basis to the spinorbital one (for example using the functions defined by Michael in the tools.py file).
EOM code's documentation must clearly state in what format the electron integrlas are expected to be inputted, being the users responsibility to provide the right input.

**Comment by msricher** [comment]
Yeah, maybe some explicit documentation on exactly which integrals we want would be nice.  Specifically, it is physicist-notation, generalized, antisymmetric, molecular integrals.

**Comment by msricher** [comment]
I think you can reproduce one of the tables from Helgaker, or Sherrill's notes, which give the definition of integrals,
<i|j> = \int{...}
<ii|jj> = \int{...}
<ii||jj> = \int{...}
etc.

**Comment by msricher** [comment]
Actually, after thinking a bit more..

Most programs just specify what their native format, or internal representation, is.  You can pass options if you're manually specifying integrals -- either "physicist" or "chemist" notation, and either "symmetric" or "antisymmetric" -- and then the program converts it to whichever format it uses natively.

**Comment by gabrielasd** [comment]
Yes, that is a good idea, thanks! :D
